storage/storage_image.go

```diff
@@ -341,8 +341,11 @@ func (s *storageImageSource) GetSignatures(ctx context.Context, instanceDigest *
 
 // newImageDestination sets us up to write a new image, caching blobs in a temporary directory until
 // it's time to Commit() the image
-func newImageDestination(imageRef storageReference) (*storageImageDestination, error) {
-	directory, err := ioutil.TempDir(tmpdir.TemporaryDirectoryForBigFiles(), "storage")
+func newImageDestination(imageRef storageReference, sys *types.SystemContext) (*storageImageDestination, error) {
```

Comment:
By convention, the parameters go `context.Context, *types.SystemContext`, then others.